src/utils/CommonSchema.ts

```diff
@@ -0,0 +1,114 @@
+/* tslint:disable */
+const Joi = require("@hapi/joi")
+  .extend(require("@hapi/joi-date"));
+
+import {
+  approvalType,
+  bodyTypeDescription, euVehicleCategory, fitmentCode,
+  fuelPropulsionSystem, microfilmDocumentType, plateReasonForIssue, vehicleClassDescription,
+  vehicleConfiguration, vehicleType, recordCompleteness
+} from "./ValidationUtils";
+
+export const brakesSchema = Joi.object().keys({
+  dtpNumber: Joi.string().max(6).required(),
```

Comment:
the brakes schema seems incomplete. Or you need to validate only one field?

Reply:
that is the common schema. It means that for the brakes, PSV, HGV and TRL have in common only the dtpNumber attribute. The other attributes are added when the common schema is extended for each vehicle (see psvValidations.ts where I extend the common schema and add the other attributes)

Reply:
Yes, I saw that, but all the fields in the `brakes` section are mandatory. I just wanted to make sure they weren't missed